filter/filter_impl/generic_filter.go

```diff
@@ -84,84 +78,3 @@ func (ef *GenericFilter) OnResponse(_ context.Context, result protocol.Result, _
 func GetGenericFilter() filter.Filter {
 	return &GenericFilter{}
 }
-
-func struct2MapAll(obj interface{}) interface{} {
```

Comment:
Maybe the removal of this function caused ci to fail

Reply:
Fixed.